Dockerfile

```diff
@@ -1,60 +1,50 @@
 ###########
 # BUILDER #
 ###########
-FROM python:3.8.1-slim as builder
-
-WORKDIR /usr/src/app
-
-# Set build variables
-ENV PYTHONDONTWRITEBYTECODE=1
-ENV PYTHONUNBUFFERED=1
-
-RUN apt-get update &&                                               \
-    apt-get upgrade -y &&                                           \
-    apt-get install -y --no-install-recommends autoconf automake    \
-	build-essential gcc libbz2-dev libcairo2 libcurl4-gnutls-dev    \
-	libffi-dev libgdk-pixbuf2.0-0 liblzma-dev libpango-1.0-0        \
-	libpangocairo-1.0-0 libssl-dev make python3-cffi python3-dev    \
-	python3-pip python3-wheel shared-mime-info zlib1g-dev           \
-	openssl ca-certificates gcc wget git
-
-# Copy app
-COPY . /usr/src/app
-RUN pip install --no-cache-dir --upgrade pip &&           \
-    pip wheel --no-cache-dir --no-deps     \
-        --wheel-dir /usr/src/app/wheels    \
-		Cython gunicorn  &&                \
-    pip wheel --no-cache-dir               \
-        --wheel-dir /usr/src/app/wheels    \
-        --editable .[coverage]
+FROM clinicalgenomics/python3.8-cyvcf2-venv:1.0 AS python-builder
+
+ENV PATH="/venv/bin:$PATH"
+
+WORKDIR /app
+
+# Install Scout dependencies
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt gunicorn
 
 
 #########
 # FINAL #
 #########
-
-FROM python:3.8.1-slim
+FROM python:3.8-slim
 
 LABEL about.home="https://github.com/Clinical-Genomics/scout"
 LABEL about.documentation="https://clinical-genomics.github.io/scout"
 LABEL about.tags="WGS,WES,Rare diseases,VCF,variants,SNP,Next generation sequencing"
 LABEL about.license="MIT License (MIT)"
 
-# Run app on non-root user
-RUN useradd -m worker && mkdir -p /home/worker/app
-WORKDIR /home/worker/app
+# Install base dependencies
+RUN apt-get update && \
+     apt-get -y upgrade && \
+     apt-get -y install -y --no-install-recommends libpango-1.0-0 libpangocairo-1.0-0 && \
```

Comment:
Much cleaner like this - nice with the builder base!

Reply:
Thanks for the review! 🙏🏻